CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -249,4 +252,12 @@ public double getRobotPitch() {
     public double getRobotRoll() {
         return imu.getRoll();
     }
+    
+    public DefenseState getDefenseState() {
+        return this.defenseState;
+    }
+    
+    public void setDefenseState(DefenseState defenseState) {
+        this.defenseState = defenseState;
```

Comment:
This would be a great place for a logging latch - a state that doesn't update all that often (we hope).